public/stylesheets/app/_modal_popup.scss

```diff
@@ -0,0 +1,60 @@
+body{
+  width: 100%;
+}
+
+.modal-container {
+  display: none; /* Hidden by default */
+  position: fixed; /* Stay in place */
+  z-index: 1; /* Sit on top */
+  left: 0;
+  top: 0;
+  width: 100%; /* Full width */
+  height: 100%; /* Full height */
+  overflow: auto; /* Enable scroll if needed */
+  background-color: rgb(0,0,0); /* Fallback color */
+  background-color: rgba(0,0,0,0.4); /* Black w/ opacity */
+}
+
+.modal-close {
+  float: right!important;
```

Comment:
should there be a space before the `!important`? This might be a Johny thing and not a standards thing

Reply:
Also could we use flex box or grid layout to put it on the right? rather than floats?

Reply:
I'll give it a go 🤞